app/src/main/java/com/eveningoutpost/dexdrip/Models/DateUtil.java

```diff
@@ -42,6 +42,21 @@ public static Date tolerantFromISODateString(String isoDateString) {
         }
     }
 
+    /**
+     * Takes in an ISO date string of the following format:
+     * yyyy-MM-dd'T'HH:mm:ss.SSS(Z|+/-HH:mm) and parses it to a Java Date object.
+     *
+     * The milliseconds and time zone/offset values are optional.
+     *
+     * @param isoDateString the iso date string to parse
+     * @param omitMilliseconds whether milliseconds should be returned or ignored
+     * @return the date object
+     */
+    public static Date tolerantFromISODateString(String isoDateString, boolean omitMilliseconds) {
```

Comment:
The use of `omitMilliseconds` parameter isn't implemented here which makes this method signature confusing. It should fall back to the existing method when `false`